src/CMakeLists.txt

```diff
@@ -37,18 +37,14 @@ ENDIF(UNIX)
 
 #This project will create many plugins as shared libraries, listed here
 SET(plugins
-  joint-torque-controller
-  force-torque-estimator
-  joint-trajectory-generator
-  nd-trajectory-generator
-  se3-trajectory-generator
-  inverse-dynamics-controller
+#  joint-torque-controller
```

Comment:
It is a temporary state. Currently the pull request compiles only
inverse-dynamics-balance-controller
position-controller
control-manager
free-flyer-locator
Because they are the only ones which have been translated (yes, I am late...)